docs/components/DocumentationFooter.js

```diff
@@ -33,10 +33,11 @@ function githubUrl(path) {
     }
   }
 
-  let pathAsMarkdown = path.replace(/\/$/, '') + '.md';
+  let pathAsMarkdown = path.replace(/\/$|\/#__next$/, '') + '.md';
```

Comment:
This looks overly precise -- our intent is to get the pathname without any query strings or fragments, not just `#__next`. I think we may want to figure out why `path` has a fragment in the first place and fix the issue at the source (i.e. an assertion that `path` doesn't have `?` or `#` always should pass).